nvm.sh

```diff
@@ -1235,6 +1235,8 @@ nvm_get_arch() {
     EXIT_CODE=$?
     if [ $EXIT_CODE -ne 0 ]; then
       HOST_ARCH=$(isainfo -n)
+    else
+      HOST_ARCH=$(echo "$HOST_ARCH" | tail -1)
```

Comment:
That's what I was going for originally, but then it would mean that the `EXIT_CODE` value above would represent the exit code of  `tail -1`, which would always be `0`, and the rest of the current logic would be broken.